src/main/java/gregtech/common/tileentities/machines/multi/MTEMultiFurnace.java

```diff
@@ -258,12 +262,8 @@ public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack a
 
         if (mMaintenanceHatches.size() != 1) return false;
 
-        if (getCoilLevel().getHeat() < 9000) {
-            this.mLevel = 8 * getCoilLevel().getLevel();
-        } else {
-            this.mLevel = 1 << (getCoilLevel().getTier());
-        }
-        this.mCostDiscount = getCoilLevel().getCostDiscount();
+        this.mLevel = 4 << (getCoilLevel().ordinal() - 1);
```

Comment:
```suggestion
this.mLevel = 4 * (1 << getCoilLevel().getTier());
````

Can you change this back to getTier()? From the looks of it, this will give twice as many parallels as it should.

Reply:
`getTier` would actually be incorrect to use in this case since it considers LV as tier 0 instead of 1. The current code works as intended.
```java
    /**
     * @return the coil tier, used for discount in the Pyrolyse Oven for example. LV == 0
     */
    public byte getTier() {
        return (byte) (this.ordinal() - 2);
    }
 ```
 Cupro
 
![image](https://github.com/user-attachments/assets/40c9bf1d-b460-42ba-941c-45a71e08592a)

Eternal

![image](https://github.com/user-attachments/assets/b5f18e4a-947b-4b3c-af48-eadbab5d0dea)

Reply:
Additionally, it might be more intuitive to match the formula as presented in game and use the tier shown in the NEI preview.

![image](https://github.com/user-attachments/assets/72a3c81f-ca26-42b1-ab26-084d72726a0a)